### PR TITLE
Update graphics.rst

### PR DESCRIPTION
**Pull request by Lyokanthrope**
Adding link to a kernel patch I was shown to help fix graphics tests on an aarch64 device. Might be useful to share.

### CODE REVIEW

**Comment by UniversalSuperBox** [comment]
image

Those underlines always cause problems

**Comment by Lyokanthrope** [comment]
So wait is that actually a problem or is that just Travis being silly?

**Comment by LNJ2** [comment]
@Lyokanthrope There are just too few hyphens in the line 67 (it looks awful in a text editor).

File: porting/debug-build/graphics.rst [hunk 1]
```diff
@@ -60,3 +60,12 @@ This error may appear in :ref:`logcat` when running these tests::
...
 To resolve this, toggle ``CONFIG_FW_***LOADER`` in your kernel config.
+
+
+
+test_hwcomposer failure on aarch64 device with armhf rootfs
+-----------------------------------------------------------
+
+Given that the current rootfs' available are all armhf, test_hwcomposer may simply fail despite doing the above on 64-bit devices.
+
+As per Ivan on Telegram, please apply `this patch <https://github.com/Halium/android_kernel_oneplus_msm8996/pull/1/commits/3e019b8700b36aef379846a748e3a447a8c3dbe7>`_  to your kernel repository
```
Comment (vanyasem) [inline porting/debug-build/graphics.rst]: It's not "per Ivan on Telegram", this patch was required long before I joined the project ;). Though I appreciate that :P

File: porting/debug-build/graphics.rst [hunk 2]
```diff
@@ -60,3 +60,12 @@ This error may appear in :ref:`logcat` when running these tests::
    kgsl kgsl-3d0: |_load_firmware| request_firmware(a330_pm4.fw) failed: -2 … kgsl kgsl-3d0: |adreno_init| Reading pm4 microcode failed a330_pm4.fw
 
 To resolve this, toggle ``CONFIG_FW_***LOADER`` in your kernel config.
+
+
+
+test_hwcomposer failure on aarch64 device with armhf rootfs
+-----------------------------------------------------------
+
+Given that the current rootfs' available are all armhf, test_hwcomposer may simply fail despite doing the above on 64-bit devices.
```
Comment (vanyasem) [inline porting/debug-build/graphics.rst]: That's not true, there is an experimental aarch64 UBports rootfs and an experimental aarch64 Arch Linux rootfs by me

File: porting/debug-build/graphics.rst [hunk 3]
```diff
@@ -60,3 +60,12 @@ This error may appear in :ref:`logcat` when running these tests::
    kgsl kgsl-3d0: |_load_firmware| request_firmware(a330_pm4.fw) failed: -2 … kgsl kgsl-3d0: |adreno_init| Reading pm4 microcode failed a330_pm4.fw
 
 To resolve this, toggle ``CONFIG_FW_***LOADER`` in your kernel config.
+
+
+
+test_hwcomposer failure on aarch64 device with armhf rootfs
+-----------------------------------------------------------
+
+On an armhf only rootfs, test_hwcomposer may simply fail despite doing the above on 64-bit devices.
```
Comment (vanyasem) [inline porting/debug-build/graphics.rst]: Actually it *will* always fail. iirc only 3.18 kernels are affected

**Comment by vanyasem** [comment]
I'll merge the PR now, if kernels other than 3.18 are affected, I will just make an another one

**Pull request merged**